.gitattributes

```diff
@@ -1,18 +1,19 @@
 # Auto detect text files and perform LF normalization
 *        text=auto
 
-*.java   text
-*.html   text
-*.kt     text
-*.kts    text
-*.md     text diff=markdown
+*.java   text eol=lf
```

Comment:
@sdelamo I can't commit my changes without this, otherwise build will be failed.

Now there is such a problem: if I leave this file as it is, then locally I cannot build the project. If I run SpotlesApply the project is built locally but not built on your СI-СD. My settings fix this problem, but it's time for you to fix the template itself so that you don't have these problems in the future.

Reply:
can you send a PR to [micronaut-project-template](https://github.com/micronaut-projects/micronaut-project-template/blob/master/.gitattributes) explain why this change is necessary?

Reply:
> I can't commit my changes without this, otherwise build will be failed.
> 
> Now there is such a problem: if I leave this file as it is, then locally I cannot build the project. If I run SpotlesApply the project is built locally but not built on your СI-СD. My settings fix this problem, but it's time for you to fix the template itself so that I and other Windows users don't have these problems in the future.

Is the issue that current `.gitattributes` is incompatible for Windows users? I don't understand what you say about CI. Currently, CI builds fine with our current `.gitattributes`

Reply:
@sdelamo done: https://github.com/micronaut-projects/micronaut-project-template/pull/348

Reply:
@sdelamo reverted this file